### PR TITLE
ABC

### PR DESCRIPTION
**Pull request by haipq7641**

### CODE REVIEW

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 1]
```diff
@@ -12,13 +12,19 @@ protected void onCreate(Bundle savedInstanceState) {
         // test
     }
 
-    private void XXXXXheckV1()      {
 
-        String   ABC = "ádsdsd";
+
+    private void XXXheckV1()          {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L17
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 2]
```diff
@@ -12,13 +12,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
+        String   ABC   = "áds dsd";
             int count   = 5;
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L20
 

&apos;member def type&apos; have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L20
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 3]
```diff
@@ -12,13 +12,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
 
         boolean isOK = true;
 
             }
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L25
 

&apos;method def rcurly&apos; have incorrect indentation level 12, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 4]
```diff
@@ -12,13 +12,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
+    public void sdsfdsfdsf (){
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L27
 

&apos;(&apos; is preceded with whitespace.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 5]
```diff
@@ -12,13 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
         // test
     }
 
-    private void XXXXXheckV1()      {
-
-        String   ABC = "ádsdsd";
-            int count   = 5;
+        private void OkDone() {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L15
 

&apos;method def modifier&apos; have incorrect indentation level 8, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L15
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 6]
```diff
@@ -12,13 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
+    }
 
-        boolean isOK = true;
+    private void ANHSONDEPTRAI() {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L19
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 7]
```diff
@@ -12,13 +12,7 @@ protected void onCreate(Bundle savedInstanceState) {
...
-    private void XXXXXheckV1()      {
-
-        String   ABC = "ádsdsd";
-            int count   = 5;
-
-
-        boolean isOK = true;
-
-            }
+    private void Test_beTa() {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L15
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 8]
```diff
@@ -12,13 +12,7 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                        final int ok = 1;
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L16
 

&apos;member def modifier&apos; have incorrect indentation level 24, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L16
 

&apos;method def&apos; child have incorrect indentation level 24, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 9]
```diff
@@ -12,13 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
...
     }
 
-    private void XXXXXheckV1()      {
-
-        String   ABC = "ádsdsd";
-            int count   = 5;
-
-
-        boolean isOK = true;
+    private void Test_beTa() {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L15
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 10]
```diff
@@ -12,13 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                        final int ok = 1;
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L16
 

&apos;member def modifier&apos; have incorrect indentation level 24, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L16
 

&apos;method def&apos; child have incorrect indentation level 24, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 11]
```diff
@@ -12,13 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    }
 
-            }
+    private void            SON_DEP_TRAI()     {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L19
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 12]
```diff
@@ -12,13 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        // Vai LEU
+                            }
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L21
 

&apos;method def rcurly&apos; have incorrect indentation level 28, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 11 above)
Comment (haipq7641) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Test OK

**Review by haipq7641 — COMMENTED** [review]
test

**Pull request merged**